src/main/java/com/samoilenko/onlinebookstore/dto/bookdtos/BookRequestDto.java

```diff
@@ -19,4 +20,5 @@ public class BookRequestDto {
     private String isbn;
     private String description;
     private String coverImage;
+    private Set<Long> categoryIds;
```

Comment:
U can provide some validation here also, for positive or zero id`s or not empty set

Reply:
I think it is not necessary because request will be created on front and those type validation need to be there